### ISSUE TITLE
pnnx converting model has permute layer missing

### ISSUE THREAD

**Issue by AnanasPizzaMigliore** [Tencent/ncnn#5679]
## model | 模型 | モデル
1. original model

## how to reproduce | 复现步骤 | 再現方法
1.last layer in ncnn param ConvolutionDepthWise     convdwrelu_14            1 1 333 352 0=128 1=5 -23310=1,1.000000e-01 11=5 12=1 13=1 14=2 2=1 3=1 4=2 5=1 6=3200 7=128 9=2
Convolution              convrelu_35              1 1 352 353 0=128 1=1 -23310=1,1.000000e-01 11=1 12=1 13=1 14=0 2=1 3=1 4=0 5=1 6=16384 9=2
ConvolutionDepthWise     convdwrelu_15            1 1 353 354 0=128 1=5 -23310=1,1.000000e-01 11=5 12=1 13=1 14=2 2=1 3=1 4=2 5=1 6=3200 7=128 9=2
Convolution              convrelu_36              1 1 354 355 0=128 1=1 -23310=1,1.000000e-01 11=1 12=1 13=1 14=0 2=1 3=1 4=0 5=1 6=16384 9=2
Convolution              conv_94                  1 1 355 356 0=33 1=1 11=1 12=1 13=1 14=0 2=1 3=1 4=0 5=1 6=4224
Reshape                  flatten_171              1 1 356 357 0=-1 1=33
Concat                   cat_12                   4 1 339 345 351 357 out0 0=1
2.last layers in pnnx param nn.LeakyReLU             head.cls_convs.3.1.act   1 1 274 275 negative_slope=1.000000e-01 #274=(1,128,20,13)f32 #275=(1,128,20,13)f32
nn.Conv2d                convbn2d_113             1 1 275 276 bias=True dilation=(1,1) groups=1 in_channels=128 kernel_size=(1,1) out_channels=128 padding=(0,0) padding_mode=zeros stride=(1,1) @bias=(128)f32 @weight=(128,128,1,1)f32 $input=275 #275=(1,128,20,13)f32 #276=(1,128,20,13)f32
nn.LeakyReLU             pnnx_unique_12           1 1 276 277 negative_slope=1.000000e-01 #276=(1,128,20,13)f32 #277=(1,128,20,13)f32
nn.Conv2d                head.gfl_cls.3           1 1 277 278 bias=True dilation=(1,1) groups=1 in_channels=128 kernel_size=(1,1) out_channels=33 padding=(0,0) padding_mode=zeros stride=(1,1) @bias=(33)f32 @weight=(33,128,1,1)f32 #277=(1,128,20,13)f32 #278=(1,33,20,13)f32
torch.flatten            torch.flatten_17         1 1 278 279 end_dim=-1 start_dim=2 $input=278 #278=(1,33,20,13)f32 #279=(1,33,260)f32
torch.cat                torch.cat_13             4 1 249 259 269 279 280 dim=2 #249=(1,33,16000)f32 #259=(1,33,4000)f32 #269=(1,33,1000)f32 #279=(1,33,260)f32 #280=(1,33,21260)f32
Tensor.permute           Tensor.permute_0         1 1 280 281 dims=(0,2,1) $input=280 #280=(1,33,21260)f32 #281=(1,21260,33)f32
pnnx.Output              pnnx_output_0            1 0 281 #281=(1,21260,33)f32
3.

**Comment by nihui** [comment]
https://github.com/Tencent/ncnn/pull/6308

**Comment by nihui** [comment]
fixed in https://github.com/Tencent/ncnn/commit/be6972b96ae2c0ffdd0fa34fbdd06b7b8e050196
